packaging/cpackconfig_common.cmake

```diff
@@ -1,13 +1,19 @@
 cpack_add_component(_cura
-                    DISPLAY_NAME "Ultimaker Cura"
-                    Description "Ultimaker Cura Executable and Data Files"
-                    REQUIRED
+		DISPLAY_NAME "Ultimaker Cura ${CURA_BUILDTYPE}"
+		Description "Ultimaker Cura ${CURA_BUILDTYPE} Executable and Data Files"
+		REQUIRED
```

Comment:
Mind that this file is using a mix of tabs and spaces now. Our code style conventions dictate to indent with 4 spaces.

Reply:
Sorry @Ghostkeeper, I was not aware of that. I replaced the tabs with 4 spaces as indicated, It should be fine now :)